commit 362ddc4efc0221e8128adc01ea891f083423ca91 (from 58cad83253747426cba58c9bbbb3a55cd6e305c9)
Merge: 58cad83 e11e072
Author: ErikZalm <erik@vdzalm.eu>
Date:   Sat Apr 28 13:19:19 2012 -0700

    Merge pull request #155 from Atntias/patch-1
    
    adding M120 and 121 for turning on and off the endstops only for homing

diff --git a/Marlin/Marlin.pde b/Marlin/Marlin.pde
index a7c4c11..d91e355 100644
--- a/Marlin/Marlin.pde
+++ b/Marlin/Marlin.pde
@@ -1100,6 +1100,12 @@ void process_commands()
       
       SERIAL_PROTOCOLLN("");
       break;
+    case 120: // M120
+      enable_endstops(false) ;
+      break;
+    case 121: // M121
+      enable_endstops(true) ;
+      break;
     case 119: // M119
       #if (X_MIN_PIN > -1)
         SERIAL_PROTOCOLPGM(MSG_X_MIN);

commit 362ddc4efc0221e8128adc01ea891f083423ca91 (from e11e072eca83ec72863bff98691914356288436c)
Merge: 58cad83 e11e072
Author: ErikZalm <erik@vdzalm.eu>
Date:   Sat Apr 28 13:19:19 2012 -0700

    Merge pull request #155 from Atntias/patch-1
    
    adding M120 and 121 for turning on and off the endstops only for homing

diff --git a/Marlin/stepper.cpp b/Marlin/stepper.cpp
index 29be5c6..85e9bbb 100644
--- a/Marlin/stepper.cpp
+++ b/Marlin/stepper.cpp
@@ -85,7 +85,7 @@ volatile char count_direction[NUM_AXIS] = { 1, 1, 1, 1};
 #ifdef ENDSTOPS_ONLY_FOR_HOMING
   #define CHECK_ENDSTOPS  if(check_endstops)
 #else
-  #define CHECK_ENDSTOPS
+  #define CHECK_ENDSTOPS  if(check_endstops)
 #endif
 
 // intRes = intIn1 * intIn2 >> 16

